src/basic_recipes/text.jl

```diff
@@ -359,7 +383,8 @@ function apply_lineheight!(lines, lh)
     for (i, line) in enumerate(lines)
         for j in eachindex(line)
             l = line[j]
-            l = Setfield.@set l.origin[2] -= (i-1) * 20 # TODO: Lineheight
+            ox, oy = l.origin
```

Comment:
Should keep the todo comment here as it seems there's a hardcoded line height still?